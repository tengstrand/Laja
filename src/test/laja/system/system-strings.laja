
#set (a = null)
#set (b = "")
#set (c = "abc")

#write "target/test-output/system-strings.out"
  strings.isEmpty(a) = {strings.isEmpty(a)}
  strings.isEmpty(b) = {strings.isEmpty(b)}
  strings.isEmpty(c) = {strings.isEmpty(c)}
#end
