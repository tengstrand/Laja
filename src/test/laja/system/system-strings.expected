  strings.isEmpty(a) = true
  strings.isEmpty(b) = true
  strings.isEmpty(c) = false
